lib/internal/modules/esm/get_format.js

```diff
@@ -93,6 +93,10 @@ function getFileProtocolModuleFormat(url, context = { __proto__: null }, ignoreE
   const { source } = context;
   const ext = extname(url);
 
+  if (ext === '.ts') {
```

Comment:
what happens if the user uses `.cts` or `.mts`?

Reply:
I dont think we should support those two formats initially and treat everything as esm

Reply:
Supporting `.mts` seems like a low-hanging fruit.